src/styles/components/_definition.scss

```diff
@@ -1,18 +1,56 @@
 $breakpoint-background-image-size: 75rem;
 
 .cmp-definition {
-  position: relative;
   line-height: 1.75rem;
   color: var(--foreground-color);
+  width: 100%;
+  position: relative;
+  margin: 2.5rem 0 0; // top: 40px; left&right: 0; bottom: 0;
+
+  @media (min-width: $breakpoint-scale-up) {
+    margin-bottom: -8.5625rem; // compensates for code example padding
+    background:
+      linear-gradient(to right, var(--background-color) 50%, transparent),
```

Comment:
The 50% value here doesn't fit the design at most screen sizes, either fading too late (empty black space) or too soon (opaque over white text). Could you try lowering it slightly in the media queries for larger screen sizes to get it closer?

Reply:
Per our conversation, I didn't make all of the changes you were able to make on the other branch. I did try to fix this linear-gradient though. Is this better? https://github.com/sparkbox/accessible-components/pull/89/commits/b66678044a96af3bcde6ada172b7ac1c90938d69

Reply:
Right, sorry for not including a screenshot. The slight adjustment in b666780 helped a lot--it fixes the fade in I thought was happening a bit too late, and honestly I think I exaggerated how different it was from the design in the first place 😓 . Apologies for the confusion!